src/maliput_malidrive/base/road_geometry.h

```diff
@@ -96,32 +96,69 @@ class RoadGeometry final : public maliput::geometry_base::RoadGeometry {
   /// @throw maliput::common::assertion_error When there is no a function described for `road_id`.
   const road_curve::Function* GetReferenceLineOffset(const xodr::RoadHeader::Id& road_id) const;
 
+  /// @brief Represents a OSC-XML Lane Position.
+  /// See
+  /// https://publications.pages.asam.net/standards/ASAM_OpenSCENARIO/ASAM_OpenSCENARIO_XML/latest/generated/content/LanePosition.html
+  struct OpenScenarioLanePosition {
+    /// The id of the road in the OpenDrive file.
+    int road_id;
+    /// The s-coordinate taken along the road's reference line from the start point of the target road.
+    double s;
+    /// The lane id.
+    int lane_id;
+    /// The lateral offset to the center line of the target lane.
+    double offset;
+  };
+
   /// Converts an OpenScenario LanePosition to a maliput RoadPosition.
   /// See
   /// https://publications.pages.asam.net/standards/ASAM_OpenSCENARIO/ASAM_OpenSCENARIO_XML/latest/generated/content/LanePosition.html
   ///
-  /// @param xodr_road_id The id of the road in the OpenDrive file.
-  /// @param xodr_s The s-coordinate taken along the road's reference line from the start point of the target road.
-  /// @param xodr_lane_id The lane id.
-  /// @param offset The lateral offset to the center line of the target lane.
   /// @returns A maliput RoadPosition.
   ///
   /// @throws When the correspondent maliput lane is not found.
-  maliput::api::RoadPosition OpenScenarioLanePositionToMaliputRoadPosition(int xodr_road_id, double xodr_s,
-                                                                           int xodr_lane_id, double offset) const;
+  maliput::api::RoadPosition OpenScenarioLanePositionToMaliputRoadPosition(
+      const OpenScenarioLanePosition& xodr_lane_position) const;
+
+  /// Converts a maliput RoadPosition to an OpenScenario LanePosition.
+  /// See
+  /// https://publications.pages.asam.net/standards/ASAM_OpenSCENARIO/ASAM_OpenSCENARIO_XML/latest/generated/content/LanePosition.html
+  ///
+  /// @param road_position The road position to get the OS lane position from.
+  /// @returns An Open Scenario LanePosition.
+  ///
+  /// @throws When the correspondent open scenario road is not found.
+  OpenScenarioLanePosition MaliputRoadPositionToOpenScenarioLanePosition(
+      const maliput::api::RoadPosition& road_position) const;
+
+  /// @brief Represents a OSC-XML Road Position.
+  /// See
+  /// https://publications.pages.asam.net/standards/ASAM_OpenSCENARIO/ASAM_OpenSCENARIO_XML/latest/generated/content/RoadPosition.html
+  struct OpenScenarioRoadPosition {
+    /// The id of the road in the OpenDrive file.
+    int road_id;
+    /// The s-coordinate taken along the road's reference line from the start point of the target road.
+    double s;
+    /// The t-coordinate taken on the axis orthogonal to the reference line of the road.
+    double t;
```

Comment:
nit: move this struct close to the OpenScenarioLanePosition one.
Also to be consistent with the google style guide we are using move this structs before the definitions of the methods.